engine/classes/Elgg/Notifications/Event.php

```diff
@@ -1,12 +1,14 @@
 <?php
 namespace Elgg\Notifications;
+
 /**
- * Notification event
+ * Subscription notification event
  * 
  * @package    Elgg.Core
  * @subpackage Notifications
  */
-class Event {
+class Event implements NotificationEvent {
```

Comment:
@mrclay Any way to make this into a `SubscriptionNotificationEvent` without breaking BC?